### ISSUE TITLE
how to prepare data for caffe neural network

### ISSUE THREAD

**Issue by Benedetta89** [BVLC/caffe#3627]
Hi, i'm new to caffe. I have to create a deep net for a classification problem but i don't understand how i have to prepare my data to put them into the net. I have 20 jpeg images for training and 20 jpeg images for test of size 584x565. Now starting from those images i have to obtain subimages of size 19x19, each one with the respective label (obtained observing the values of the pixel in the subimages 19x19). what can i do this tranformation in caffe ? Do the images have to be transormed into blobs or what else?). Once obtained this subimages , they  rappresent (whit their labels) my training set (test set),but how format they have to be to give them to the net?
Thanks for help

**Comment by seanbell** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
